app/views/placements/support/_secondary_navigation.html.erb

```diff
@@ -0,0 +1,7 @@
+<%= render SecondaryNavigationComponent.new do |component| %>
+  <% component.with_navigation_item t(".details"), organisation_url(organisation) %>
+  <% component.with_navigation_item t(".users"), "#" %>
```

Comment:
Could you put in the users paths.
You might need a helper for this, similar to what you've done above.